src/Appwrite/Network/Validator/Host.php

```diff
@@ -54,7 +54,7 @@ public function getDescription()
      */
     public function getType()
     {
-        return self::TYPE_STRING;
+        return 'string';
```

Comment:
I think self should still work here. If not, you can try `static::`

Reply:
There's nothing to inherit in `appwrite/appwrite` right now because the type definitions are still in the utopia branch: https://github.com/utopia-php/framework/commit/0b7934ab3a1e081f612f6385fa9100fe9ba191c1

Am I understanding your feedback correctly?

Reply:
Yep, sorry. Once we release the Utopia changes we can inherit the constants properly.